docs/UserGuide.md

```diff
@@ -127,6 +128,9 @@ Some example commands you can try are:
 
 * `listing add 2 t/HDB a/Adam street, block 456, #03-03` : Adds a listing for the client with index 2 in PROperty.
 
+<div markdown="block" class="alert alert-primary">
+:bulb: Recommended Window Size<br> For the best experience, use a window size of at least 1920 x 1080 pixels.
```

Comment:
Might face some criticism by users who claim they do not know how to set their window to this specific dimensions.

Would it be better if we just tell the user to "full screen" the application?

Reply:
Maybe can say "or full screen on your laptop"?

Reply:
But if we leave the pixel dimensions in there, a user might not know how to get to that size.

Reply:
I see what you mean ok I changed it